Package/gulpfile.js

```diff
@@ -75,7 +75,7 @@ const buildTypeScript = async () => {
 
 const makeXCodeProj = async () => {
   shelljs.mkdir('-p', 'iOS/Build');
-  exec(`cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=../../../Apps/Playground/Playground/node_modules/@babylonjs/react-native-iosandroid/submodules/BabylonNative/Dependencies/ios-cmake/ios.toolchain.cmake -DPLATFORM=OS64COMBINED -DENABLE_ARC=0 -DENABLE_BITCODE=1 -DDEPLOYMENT_TARGET=12 -DENABLE_PCH=OFF ${cmakeBasekitBuildDefinition} ..`, 'iOS/Build');
+  exec(`cmake -B Build -G Xcode ${cmakeBasekitBuildDefinition}`, 'iOS');
```

Comment:
This seems like it's missing `-D IOS=ON`. How is it working without it?

Reply:
The IOS flag affects BN's top level CMakeLists.txt, but that is not the top level CMakeLists.txt for BRN. The top level ones for BRN are `Modules/@babylonjs/react-native-iosandroid/ios/CMakeLists.txt` and `Package/iOS/CMakeLists.txt`, both of which set the same cmake variables that BN's CMakeLists.txt conditionally does based on the IOS flag.